packages/schemas/src/gen/index.ts

```diff
@@ -152,7 +154,6 @@ const generate = async () => {
   // Generate DB entry types
   await Promise.all(
     generated.map(async ([file, { tables }]) => {
-      // LOG-88 Need refactor, disable mutation rules for now.
```

Comment:
no need for refactor any more?

Reply:
Remove it since I found this issue was canceled.

<img width="924" alt="image" src="https://user-images.githubusercontent.com/10594507/183561249-6a2af16d-0aa5-4d66-a564-a994138557f4.png">